src/UI/Shared/WB.UI.Shared.Enumerator/Resources/layout/interview_question__attachment.axml

```diff
@@ -42,6 +42,6 @@
             local:MvxBind="Text Localization('Interview_ShowPdf'); 
                      Click Attachment.ShowPdf; 
                      Visible Attachment.IsPdf;
-                     Tag Format('Show_Pdf_{0}', Identity)"/>
+                     Tag Format('Show_Pdf_{0}', ItemTag)"/>
```

Comment:
Need get ```Attachment.Tag``` may be. Because two pdf attachments will be with same tag now

Reply:
There is no identity for QuestionOption binding was failing for options. 
Item tag is built with respect to question identity (at least for the options). It should be OK if we are creating ItemTag correctly. Otherwise we should fix it.